tiledb/sm/config/config.h

```diff
@@ -87,6 +87,16 @@ class Config {
    */
   static const std::string CONFIG_LOGGING_LEVEL;
 
+  /** An array will open between this value and timestamp_end upon a
```

Comment:
For block style comments, the first line should be empty for consistency with (most) of the code base. For example:
```
/**
 * foo
 * bar
 */
```